spread/tests/smoke/task.yaml

```diff
@@ -0,0 +1,28 @@
+summary: PostgreSQL SNAP smoke tests
+
+execute: |
+  echo "Running smoke test..."
+  postgresql.psql -U postgres -h /tmp -c "SELECT version();"
```

Comment:
We should compare the actual version to what's set in the yaml to make sure we have updated the definition correctly. We should be able to get it with something like `yq .version /snap/postgresql/current/meta/snap.yaml` but we'll need to install yq for that.

Reply:
Done in [f2505f0](https://github.com/canonical/postgresql-snap/pull/31/commits/f2505f09957259f27473e2dced2764cff53e3ac1).

P.S. for some reason yq has permission issues in spread:
```
craft:ubuntu-22.04 .../spread/tests/smoke# ls -la /root/proj/snap/snapcraft.yaml
-rw-rw-r-- 1 ubuntu ubuntu 5107 Jul  3 21:03 /root/proj/snap/snapcraft.yaml

craft:ubuntu-22.04 .../spread/tests/smoke# yq .version /root/proj/snap/snapcraft.yaml
Error: open /root/proj/snap/snapcraft.yaml: permission denied

craft:ubuntu-22.04 .../spread/tests/smoke# id
uid=0(root) gid=0(root) groups=0(root)

craft:ubuntu-22.04 .../spread/tests/smoke# awk -F\' '/version/{print $2}' /root/proj/snap/snapcraft.yaml
16.9
```